src/data/fin.lean

```diff
@@ -169,15 +182,15 @@ end bit
 lemma of_nat_eq_coe (n : ℕ) (a : ℕ) : (of_nat a : fin (n+1)) = a :=
 begin
   induction a with a ih, { refl },
-  ext, show (a+1) % (n+1) = fin.val (a+1 : fin (n+1)),
+  ext, show (a+1) % (n+1) = subtype.val (a+1 : fin (n+1)),
   { rw [val_add, ← ih, of_nat],
     exact add_mod _ _ _ }
 end
 
 /-- Converting an in-range number to `fin (n + 1)` produces a result
 whose value is the original number.  -/
 lemma coe_val_of_lt {n : ℕ} {a : ℕ} (h : a < n + 1) :
-  (a : fin (n + 1)).val = a :=
+  ((a : fin (n + 1)) : ℕ) = a :=
```

Comment:
This change makes `coe_val_of_lt` the same as `coe_coe_of_lt`, meaning one of those two lemmas can be removed.

Reply:
I undid this change. It might be useful to keep the lemma.